style.css

```diff
@@ -0,0 +1,989 @@
+* {
+  font-family: 'Roboto', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+nav {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  display: flex;
+  background-color: black;
+  justify-content: space-between;
+  z-index: 10;
+}
+
+nav ul,
+footer ul {
+  display: flex;
+  list-style-type: none;
+  margin: 0 10px;
+}
+
+nav li,
+footer li {
+  color: rgb(163, 162, 162);
+  margin: 8px;
+  font-size: 13px;
+}
+
+.nav-2 li {
+  color: rgb(163, 162, 162);
+  margin: 15px 10px;
+  font-size: 14px;
+}
+
+.f-title li {
+  color: #f42;
+  padding: 0 10px;
+}
+
+.group-list li {
+  color: #f42;
+  font-size: 12px;
+  border-bottom: 1px solid #eee;
+  padding: 8px;
+}
+
+nav li:hover,
+footer li:hover {
+  color: white;
+}
+
+.nav-2 {
+  margin: 31px 0 0 0;
+  display: flex;
+  justify-content: space-between;
+  background-color: rgb(250, 250, 250);
+  position: sticky;
+  z-index: 100;
+  top: 0;
+}
+
+.nav-2 ul {
+  display: flex;
+  list-style-type: none;
+  margin: 0 5px;
+}
+
+.nav-2 li:hover {
+  color: black;
+}
+
+.nav-2-right {
+  margin: 0 5px;
+  display: flex;
+}
+
+.nav-2-icon {
+  color: rgb(163, 162, 162);
+  padding: 15px 15px;
+  border-right: 1px solid lightgrey;
+}
+
+.nav-2-icon-2 {
+  color: #f42;
+  padding: 15px 15px;
+  border-left: 1px solid lightgrey;
+}
+
+.nav-2-icon:hover {
+  color: black;
+}
+
+.latest-news,
+.latest-deals {
+  max-width: 1176px;
+  margin: 20px auto;
+  padding: 0 20px;
+}
+
+h2 {
+  color: #f42;
+}
+
+h2:hover {
+  color: black;
+}
+
+.latest-news-sec,
+.deal-items {
+  margin: 20px 0;
+  display: grid;
+  width: 100%;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  grid-column-gap: 10px;
+  grid-row-gap: 10px;
+}
+
+.images-head {
+  width: 100%;
+  padding-top: 50%;
+  background-size: cover;
+  background-position: center;
+}
+
+.tmw-logo {
+  width: 30%;
+  margin-bottom: 50px;
+}
+
+.current-date {
+  position: absolute;
+  color: white;
+  float: right;
+  top: 0;
+  margin-left: 560px;
+  margin-bottom: 50px;
+  width: 27%;
+}
+
+.news-title,
+.deals-title {
+  font-size: 16px;
+  margin: 5px 0;
+}
+
+.news-title:hover,
+.deals-title:hover {
+  color: #f42;
+  text-decoration: underline;
+}
+
+.news-source {
+  display: flex;
+  font-size: 13px;
+  color: rgb(163, 162, 162);
+}
+
+.deals-source {
+  display: flex;
+}
+
+.price {
+  font-size: 13px;
+  color: rgb(163, 162, 162);
+}
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `color: rgb(163, 162, 162);` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-y{
  color: rgb(163, 162, 162);
}
```
**Html file**

```
<div class="color-y">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_